script.js

```diff
@@ -1,6 +1,158 @@
 const hamburgMenu = document.querySelector('section.menu-bar');
 const menuList = document.querySelector('.nav-links');
 const closeIcon = document.querySelector('.close');
+const workSection = document.querySelector('#work-section');
+const cardSection = document.createElement('section');
+const backdrop = document.querySelector('#backdrop');
+const modals = document.querySelector('.modal');
+cardSection.id = 'card-section';
+
+const modalPopUp = [{
+  modalTitle: ['Multi Post Stories', 'keep track of hundred of components website'],
+  listOfTechnologies: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+  imageModal: 'images/feature_image.png',
+  modalDescription: `Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+   printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five 
+   centuries, but also the leap into electronic typesetting, remaining essent`,
+  SeeLive: 'See live',
+  seeSource: 'See source',
+
+}];
+
+const recentWorkContent = [{
+  recentWorkTitle: 'My Recent Works',
+  imageSource: 'images/Img-Placeholder.png',
+  ProjectDescriptionTitle: 'Multi-Post Stories',
+  projectDescription: `A daily selection of privately personalized reads;
+  no accounts or sign-ups required. has been the industry's standard dummy
+  text ever since the 1500s, when an unknown printer took a standard dummy text.`,
+  recentWorkButton: ['CSS', 'HTML', 'Bootstrap', 'Ruby', 'See project'],
+}];
+
+const cardSectionContent = [{
+  cardTitles: 'Profesional Art Printing Data',
+  cardTitles1: ['Data Dashboard Healthcare', 'Website Portfolio', 'Proffesional Art Printing Data More'],
+  cardDescription: `A daily selection of privately personalized reads;
+  no accounts or sign-ups required. Has been the industry's standard.`,
+  cardButton: ['HTML', 'Bootstrap', 'Ruby', 'See project'],
+}];
+
```

Comment:
- [ ] Based on the project requirement, you are expected to have an array of project details like so 👇🏾 

```javascript
const projects = [{
    id: 1,
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    id: 2,
    name: "Ghost game",
    description: "A Ghost FPS game built with javascript.",
    image: "link_to_image.jpg",
    technologies: ["HTML", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    id: 3,
    name: "CV Builder",
    description: "A CV builder which helps users .",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    id: 4,
    name: "Sun city",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

This means that each project is displayed in the work's sections and when the see project button is clicked, the modal popup appears using that specific project data populated with the data from the projects array. Currently, when I click the see project button for any project card, I see the same information and this is not supposed to be. Check an example of students' projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636.

Thanks to the above points, adding projects in the future will be easy. Students will only add it in one place - as another object in the projects array.